components/gfx/platform/freetype/font_template.rs

```diff
@@ -15,17 +15,18 @@ use webrender_api::NativeFontHandle;
 #[derive(Deserialize, Serialize)]
 pub struct FontTemplateData {
     // If you add members here, review the Debug impl below
-
     pub bytes: Option<Vec<u8>>,
     pub identifier: Atom,
 }
 
 impl fmt::Debug for FontTemplateData {
     fn fmt(&self, fmt: &mut fmt::Formatter) -> fmt::Result {
         fmt.debug_struct("FontTemplateData")
-           .field("bytes", &self.bytes.as_ref().map(|b| format!("[{} bytes]", b.len())))
-           .field("identifier", &self.identifier)
-           .finish()
+            .field(
+                "bytes",
+                &self.bytes.as_ref().map(|b| format!("[{} bytes]", b.len())),
+            ).field("identifier", &self.identifier)
```

Comment:
Shouldn't the next `.field` call start on a new line? oO

Reply:
I'll file a bug to rustfmt 😢

Reply:
Looks like a similar issue has been reported: https://github.com/rust-lang-nursery/rustfmt/issues/2932